devices/HumanStateProvider/HumanStateProvider.cpp

```diff
@@ -1419,10 +1419,10 @@ bool HumanStateProvider::impl::applyRpcCommand()
     }
     case rpcCommand::calibrateAllWithWorld: {
         // Check if the chose baseLink exist in the model
-        std::string baseLinkName = commandPro->baseLinkName;
-        if(!(kinDynComputations->getRobotModel().isFrameNameUsed(baseLinkName)))
+        std::string refLinkForCalibrationName = commandPro->refLinkName;
+        if(!(kinDynComputations->getRobotModel().isFrameNameUsed(refLinkForCalibrationName)))
```

Comment:
@lrapetti  Just a minor comment non-blocking comment, checking this link name in the model instance, rather than the kindyncomputations instance may be more certain check. 

Check https://github.com/robotology/human-dynamics-estimation/blob/a7f8724e968dfc7e5adc5b14a15bbd6d81735348/devices/HumanStateProvider/HumanStateProvider.cpp#L809

and 

https://github.com/robotology/human-dynamics-estimation/blob/a7f8724e968dfc7e5adc5b14a15bbd6d81735348/devices/HumanStateProvider/HumanStateProvider.cpp#L818

Reply:
If it is fine to you I would proceed merging as it is

Reply:
Sure, go ahead 😉